DESCRIPTION

```diff
@@ -4,12 +4,13 @@ Version: 0.0.0.9000
 Authors@R: c(
   person("Randi", "Garcia", email = "rgarcia@smith.edu", role=(c("aut", "cre"))),
   person("David", "Kenny", email = "david.kenny@uconn.edu", role = "aut"),
-  person("Rutendo", "Madziwo", email = "rmadziwo@smith.edu", role = "ctb"))
+  person("Rutendo", "Madziwo", email = "rmadziwo@smith.edu", role = "ctb"),
+  person("Sifan", "Liu", email = "cliu13@smith.edu", role = "ctb"))
 Description: This package provides helper functions for dyadic data analysis.
 Depends:
     R (>= 3.2.3),
     nlme, MASS
-License: Please acknowledge authors
+License: CC0
```

Comment:
@RandiLGarcia will want to weigh in on this. Is [Creative Commons](https://en.wikipedia.org/wiki/Creative_Commons_license#Zero_/_public_domain) OK with you, @RandiLGarcia ?

Reply:
This looks good to merge if it is OK with you @RandiLGarcia .

Reply:
Creative commons is fine.